meilisearch-release-changelog/v0.29.0.md

```diff
@@ -0,0 +1,134 @@
+# v0.29.0 release changelogs
+
+<!-- The following line should NOT be put in the official release changelogs -->
+⚠️ Since this is a release candidate (RC), we do NOT recommend using this version in production. This is only for test purposes. Please, feel free to [report bugs](https://github.com/meilisearch/meilisearch/issues/new/choose) and [give feedback about the new feature introductions](https://github.com/meilisearch/product/discussions).
+
+In this release, we integrated a demanded feature: requiring all query terms to be present in search results. Some users have been stuck in the pre-0.21 versions because of this.
+We also improved the indexing speed
+
+🧰 Most of the integrations (SDKs, clients, libraries) you use will be compatible with this version in the 4 next hours following the release. Sometimes it can take longer, maybe 24h, depending on some issues we can encounter during the release.
+However, here is the exhaustive list of the NON-compatible integrations with v0.29.0
+- [meilisearch-dart](https://github.com/meilisearch/meilisearch-dart) and [meilisearch-swift](https://github.com/meilisearch/meilisearch-swift): you can use it with v0.29.0 but the [`matchingStrategy` feature](#customize-matching-strategy) is not integrated yet. If you really need it, let us know, and of course, contributions are welcome!
+- [meilisearch-java](https://github.com/meilisearch/meilisearch-java): our team is still working on refactoring the code, and the SDK cannot be used with v0.29.0. Follow the repository to know when it will be compatible.
```

Comment:
```suggestion
- [meilisearch-dart](https://github.com/meilisearch/meilisearch-dart) and [meilisearch-swift](https://github.com/meilisearch/meilisearch-swift): You can use it with v0.29.0 but the [`matchingStrategy` feature](#customize-matching-strategy) is not integrated yet. Let us know if you really need it, and contributions are always welcome!
- [meilisearch-java](https://github.com/meilisearch/meilisearch-java): Our team is still working on refactoring the code, and the SDK cannot be used with v0.29.0. Follow the repository to know when it will be compatible.
```